common/src/main/java/com/wynntils/models/beacons/BeaconModel.java

```diff
@@ -149,7 +149,8 @@ private static BeaconKind beaconKindFromItemStack(ItemStack itemStack) {
             int customColor = potionContents.customColor().orElse(CommonColors.WHITE.asInt());
 
             // Log the color if it's likely to be a new beacon kind
-            if (customModel == COLOR_CUSTOM_MODEL_DATA) {
+            if (customModel == Models.Activity.BEACON_COLOR_CUSTOM_MODEL_DATA
+                    || customModel == Models.Lootrun.BEACON_COLOR_CUSTOM_MODEL_DATA) {
```

Comment:
Not a fan of this but since they don't both use the same model data anymore it'll have to do

Reply:
This is only development logging code so it matters even less.